Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY target/zaakafhandelcomponent.jar /
 RUN date -Iseconds > /build_timestamp.txt
 
 # Start zaakafhandelcomponent
-ENTRYPOINT ["java", "-jar", "zaakafhandelcomponent.jar"]
+ENTRYPOINT ["java", "-XX:+HeapDumpOnOutOfMemoryError", "-jar", "zaakafhandelcomponent.jar"]
```

Comment:
We need 1GB of disk space in the container. Best on a separate persistent volume so we can inspect this post-mortem

Reply:
I will remove this PR for now. Let's look at this later.